astropy/time/core.py

```diff
@@ -124,6 +124,16 @@ class Time(object):
     formats (e.g. JD) where very high precision (better than 64-bit precision)
     is required.
 
+    The allowed values for ``format`` and ``scale`` can be listed with::
+
+      >>> list(Time.FORMATS)
+      ['jd', 'mjd', 'decimalyear', 'unix', 'cxcsec', 'gps', 'plot_date', 'astropy_time',
+       'datetime', 'iso', 'isot', 'yday', 'fits', 'byear', 'jyear', 'byear_str',
+       'jyear_str']
+
+      >>> list(Time.SCALES)
+      ['tai', 'tcb', 'tcg', 'tdb', 'tt', 'ut1', 'utc']
```

Comment:
@taldcroft - `TIME_DELTA_SCALES` is a tuple, just like `TIME_SCALES`; the problem is that it is made from the keys of a `dict`. I would just go with my suggestion above of replacing its definition with:
```
TIME_DELTA_SCALES = (GEOCENTRIC_SCALES + BARYCENTRIC_SCALES +
                     ROTATIONAL_SCALES)
```